packages/blocks/src/__internal__/clipboard/edgeless-clipboard.ts

```diff
@@ -147,14 +147,14 @@ export class EdgelessClipboard implements Clipboard {
   }
 
   public dispose() {
-    document.body.removeEventListener('cut', this._onCut);
-    document.body.removeEventListener('copy', this._onCopy);
-    document.body.removeEventListener('paste', this._onPaste);
+    document.body.removeEventListener('cut', e => void this._onCut(e));
+    document.body.removeEventListener('copy', e => void this._onCopy(e));
+    document.body.removeEventListener('paste', e => void this._onPaste(e));
```

Comment:
This usage is wrong

Reply:
Thanks for the notification, I just thought this is part of #5035. Landing the best practice there or in new PRs should be better.

Reply:
The problem with this usage is that it is not using the exact same function reference to remove the event listener.

The arrow function used to add the event listener **creates a new function instance every time** it is called. So when trying to remove the event listener using `removeEventListener`, it will not be able to find and remove the listener because it is a different function reference.

Reply:
Yeah, you are right. I didn't notice that  they are remove event listener functions.

I'm going to extract it as a wrapper function.